python/github_com/TheThingsNetwork/api/discovery/discovery_pb2.py

```diff
@@ -7,19 +7,21 @@
 from google.protobuf import message as _message
 from google.protobuf import reflection as _reflection
 from google.protobuf import symbol_database as _symbol_database
+from google.protobuf import descriptor_pb2
 # @@protoc_insertion_point(imports)
 
 _sym_db = _symbol_database.Default()
 
 
 from google.protobuf import empty_pb2 as google_dot_protobuf_dot_empty__pb2
+ import annotations_pb2 as google_dot_api_dot_annotations__pb2
+.gogoproto import gogo_pb2 as github_dot_com_dot_gogo_dot_protobuf_dot_gogoproto_dot_gogo__pb2
```

Comment:
This is wrong

Reply:
This is generated code, not sure what you want to be fixed here.

Reply:
This is not valid python. used to delete the entire line. Now you only delete the identifiers.